core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -174,6 +174,13 @@ private TableProperties() {}
   public static final String PARQUET_BLOOM_FILTER_COLUMN_ENABLED_PREFIX =
       "write.parquet.bloom-filter-enabled.column.";
 
+  public static final String PARQUET_COLUMN_STATS_ENABLED_PREFIX =
+      "write.parquet.stats-enabled.column.";
+
+  public static final String DEFAULT_PARQUET_COLUMN_STATS_ENABLED =
+      PARQUET_COLUMN_STATS_ENABLED_PREFIX + "default";
+  public static final boolean DEFAULT_PARQUET_COLUMN_STATS_ENABLED_DEFAULT = true;
```

Comment:
how about 
- PARQUET_COLUMN_STATS_ENABLED
- PARQUET_COLUMN_STATS_ENABLED_DEFAULT

Reply:
Was trying to follow the pattern for metrics

  public static final String DEFAULT_WRITE_METRICS_MODE = "write.metadata.metrics.default";

If it is better, I can change it.

Reply:
I would vote for consistency with the bloom filter properties

Reply:
Done.